### ISSUE TITLE
package_data superfluous?

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#2914]
from http://setuptools.readthedocs.io/en/latest/setuptools.html:
```
include_package_data
    If set to True, this tells setuptools to automatically include any data files
    it finds inside your package directories that are specified by your MANIFEST.in
    file. For more information, see the section below on Including Data Files.
...
package_data
    A dictionary mapping package names to lists of glob patterns. For a complete
    description and examples, see the section below on Including Data Files.

    You do not need to use this option if you are using include_package_data,
    unless you need to add e.g. files that are generated by your setup script and
    build process. (And are therefore not in source control or are files that you don’t
    want to include in your source distribution.)
```

See "You do not need to use..." - check if that is true and we can leave it away.

**Comment by rozuur** [comment]
can I work on this?

**Comment by ThomasWaldmann** [comment]
sure. just do a PR when you need feedback on your code.

**Comment by ThomasWaldmann** [comment]
@LocutusOfBorg has a bug report that the ubuntu ("binary") borg 1.1 package is rather fat currently, contains lots of unneeded (in case of a binary package, that has the compiled *.so C extensions) .pyx, .c, .h files.

**Comment by LocutusOfBorg** [comment]
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=882201
^^ FYI

**Comment by ThomasWaldmann** [comment]
@rozuur still working on this?

**Comment by rozuur** [comment]
@ThomasWaldmann Sorry I am not able to pick this up. Please assign it to others.
Thanks

**Comment by ThomasWaldmann** [comment]
fixed by #3355.
